mlir/utils/performance/perfRunner.py

```diff
@@ -472,6 +472,7 @@ def benchmarkExternal(cls, commandLine, paths: Paths, arch, envs=dict()):
             outs, errs = p1.communicate(timeout=300)
             if len(errs) > 0:
                 print("MIOpen benchmark produced errors: ", errs.decode('utf-8'))
+                raise OSError(errs.decode('utf-8'))
```

Comment:
Timeouts are caught and ignored.  I wanted parse errors like these to cause a test failure and draw attention.  If we have other conditions that should be ignored, we can add them, but maybe should abstract the whole idiom.  Could also accumulate the "errs" strings and signal at the end of the loop if it's not empty.